src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/UpdateAzureContainerRegistry.cs

```diff
@@ -15,68 +15,60 @@
 using System.Collections;
 using System.Management.Automation;
 using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Management.ContainerRegistry.Models;
 
 namespace Microsoft.Azure.Commands.ContainerRegistry
 {
-    [Cmdlet(VerbsData.Update, ContainerRegistryNoun,
-        DefaultParameterSetName = "Empty",
-        SupportsShouldProcess = true),
-        OutputType(typeof(PSContainerRegistry))]
+    [Cmdlet(VerbsData.Update, ContainerRegistryNoun, SupportsShouldProcess = true)]
```

Comment:
@Jinhuafei the `DefaultParameterSetName` was removed

Reply:
fixed